doc/setup_instructions.md

```diff
@@ -50,11 +50,19 @@ LFS. Note that the GitHub option to download as .zip will not work
 
 You need at least Git LFS version 2.8.0, old versions do not work.
 
-On Linux use your package manager to install Git. On Mac install the
-package manager [homebrew](https://brew.sh/) if you don't already have
-it, and use it to install Git. On Mac and Linux Git LFS is likely available
-as a package named `git-lfs`. If it is not install it manually. After
-installing remember to run `git lfs install` in terminal.
+On Linux use your package manager to install Git. This can be done in the
+terminal - for instance, on Ubuntu, type: `sudo apt-get install git` . On
```

Comment:
This section is going to get really long if we start to list all the package manager commands. I'd be fine with keeping text saying something like: "This can be done in the terminal with your distro's package manager (search online for what your distro's package manager is if you don't know)"